natlas-server/README.md

```diff
@@ -56,13 +56,23 @@ Environment configs are loaded from the environment or a `.env` file and require
 | `OPENCENSUS_ENABLE` | `false` | Enables OpenCensus instrumentation to help identify performance bottlenecks. |
 | `OPENCENSUS_SAMPLE_RATE` | `1.0` | Specifies the percentage of requests that are traced with OpenCensus. A number from 0 to 1. |
 | `OPENCENSUS_AGENT` | `127.0.0.1:55678` | An OpenCensus agent or collector that this instance will emit traffic to. |
+| `MAIL_SERVER` | `None` | Mail server to use for invitations, registrations, and password resets |
+| `MAIL_PORT` | `25` | Port that `MAIL_SERVER` is listening on |
+| `MAIL_USE_TLS` | `False` | Whether or not to connect to `MAIL_SERVER` with TLS|
+| `MAIL_USERNAME` | `None` | Username (if required) to connect to `MAIL_SERVER` |
+| `MAIL_PASSWORD` | `None` | Password (if required) to connect to `MAIL_SERVER` |
+| `MAIL_FROM` | `None` | Address to be used as the "From" address for outgoing mail. This is required if `MAIL_SERVER` is set. |
+| `SERVER_NAME` | `None` | This should be set to the domain and optional port that your service will be accessed on. E.g. `natlas.io` or `10.0.0.15:5000` |
+| `PREFERRED_URL_SCHEME` | `https` | You can optionally set this value to `http` if you're not using ssl. This should be avoided for any production environments. |
 
 #### Example ENV
 
 ```text
 SECRET_KEY=Y91gWepML8Bq1IqDO7MFy9LDbtuPrS3Z9Ge6TX3a
 ELASTICSEARCH_URL=http://172.17.0.2:9200
 FLASK_ENV=production
+MAIL_SERVER=172.17.0.5
+MAIL_FROM=noreply@natlas.io
```

Comment:
Flask-Mail does support this but it might require more work from us. I just gave it a shot and this is what I saw come into the mail server lol...

```
b'Content-Type: text/plain; charset="utf-8"'
b'MIME-Version: 1.0'
b'Content-Transfer-Encoding: 7bit'
b"Subject: [Natlas] You've been invited to Natlas!"
b'From: =?utf-8?q?Natlas?= <noreply@natlas.io>'
b'To: zzzzz@actualcrimes.org'
b'Date: Mon, 22 Jun 2020 02:16:04 +0000'
b'Message-ID: <159279216402.14.11267343090365240441@01d3ce6691fd>'
b'X-Peer: 127.0.0.1'
b''
b'You\'ve been invited to Natlas! Use the link below to set a password. This link will expire in 48 hours. If you are unable to accept this invite before it expires, you can simply use the "Forgot your password?" link on the login page to set your password.'
b''
b'https://localhost:5000/auth/invite?token=Vmc4sopgYbP_jYWRRCAiSDcgdErMz4Qh4FC4TBbf29U'
b''
b'If you did not want this invitation, simply disregard it.'
------------ END MESSAGE ------------
```


of particular note is `b'From: =?utf-8?q?Natlas?= <noreply@natlas.io>'`

I'd say this deserves further investigation via a separate issue and corresponding PR, though

Reply:
Tracking this at #297 and marking this as resolved for the time being.